code/modules/research/designs/machine_designs.dm

```diff
@@ -242,6 +242,76 @@
 	build_path = /obj/item/weapon/circuitboard/rdserver
 	category = list("Research Machinery")
 
+/datum/design/gibber
+	name = "Machine Design (Gibber Board)"
+	desc = "The circuit board for a gibber."
+	id = "gibber"
+	req_tech = list("programming" = 1)
+	build_type = IMPRINTER
+	materials = list("$glass" = 1000, "sacid" = 20)
+	build_path = /obj/item/weapon/circuitboard/gibber
+	category = list ("Misc. Machinery")
+
+/datum/design/smartfridge
+	name = "Machine Design (Smartfridge Board)"
+	desc = "The circuit board for a smartfridge."
+	id = "smartfridge"
+	req_tech = list("programming" = 1)
+	build_type = IMPRINTER
+	materials = list("$glass" = 1000, "sacid" = 20)
+	build_path = /obj/item/weapon/circuitboard/smartfridge
+	category = list ("Misc. Machinery")
+
+/datum/design/monkey_recycler
+	name = "Machine Design (Monkey Recycler Board)"
+	desc = "The circuit board for a monkey recycler."
+	id = "smartfridge"
```

Comment:
This seems wrong.

Also, might want to consider making the smartfridge boards work like the vendor boards (screwdriver to change mode) so you can build the different variants.